e2e/terraform/provision-infra/compute.tf

```
@@ -2,7 +2,8 @@
 # SPDX-License-Identifier: BUSL-1.1
 
 locals {
-  ami_prefix = "nomad-e2e-v3"
+  ami_prefix           = "nomad-e2e-v3"
+  ubuntu_instance_name = "ubuntu-jammy-${var.instance_architecture}"
```

This is the image (AMI), not the instance, right?

```suggestion
  ubuntu_image_name = "ubuntu-jammy-${var.instance_architecture}"
```